spec/features/post_index_page_spec.rb

```diff
@@ -0,0 +1,120 @@
+require 'rails_helper'
+
+RSpec.describe 'Post index page', type: :feature do
```

Comment:
- [ ] 13 tests out of 54 are failing, main reason is  `Couldn't find User with 'id'=3`. I seeded db with 4 users and keep getting these errors. Please run `rspec` and make sure all tests pass
<img width="812" alt="Screenshot 2022-07-19 at 12 18 23" src="https://user-images.githubusercontent.com/77038610/179702271-e331c44f-ce8c-4e9b-87f2-c736c8ebcd0e.png">
 
If there are extra steps that we need to take to run the tests please leave comment on the extra steps to take. 


- [ ] I was getting 1 more error related to `author_id`, in `post_spec` and `user_spec` you are using `author_id` instead of `user_id`, this is failing because you do not have `author_id` column kindly use `user_id` instead

Reply:
Thanks, after the previous test, I changed the schema of my database, and I forgot to make changes in spec files as well, fixed it now.